### PR TITLE
Fixed footer broken links

### LINKED ISSUE #23771: Bug on live site
Hello developers team. On your live site https://getbootstrap.com/docs/3.3/ the footer link Examples (https://getbootstrap.com/docs/getting-started/#examples) doesn't work, as well as doesn't appear on bottom "Built with Bootstrap" previews images.

### PR DESCRIPTION
**Pull request by harishgadiya**
Fix #23771

### CODE REVIEW

**Review by XhmikosR — CHANGES_REQUESTED** [review]
You have random changes here. Revert those please.

**Comment by harishgadiya** [comment]
@XhmikosR, I have revert unwanted changes in `Gemfile.lock` file. But why travis CI is not passing the build can you please help me to fix this

**Comment by XhmikosR** [comment]
I'm still not sure against which branch this should go in. @mdo can you clear the situation a bit? We have master, v3-dev and v3.4.0-dev. Can we get rid of v3-dev after merging it to master?

@harishgadiya: The failures are because a patch we have in v3-dev isn't in master.

**Comment by harishgadiya** [comment]
Oh thanks @XhmikosR. So there's nothing from my end

File: docs/_includes/footer.html
Comment (vsn4ik) [inline docs/_includes/footer.html]: You need to delete the unnecessary line (duplicate "Examples").

**Review by vsn4ik — CHANGES_REQUESTED** [review]

File: docs/_includes/footer.html
Comment (harishgadiya) [reply docs/_includes/footer.html]: Done proposed changes

**Comment by harishgadiya** [comment]
@XhmikosR @mdo Can you please look into it. What else needs to done

**Comment by XhmikosR** [comment]
@harishgadiya: please be patient. I'm waiting for @mdo's feedback.

**Review by vsn4ik — APPROVED** [review]

**Comment by harishgadiya** [comment]
@XhmikosR @mdo did you get a chance to look into it

**Comment by XhmikosR** [comment]
https://github.com/twbs/bootstrap/pull/23808#issuecomment-326792463

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**
